Language/Haskell/GhcMod/GhcPkg.hs

```diff
@@ -87,6 +87,7 @@ getPackageDbStack = do
         getCabalPackageDbStack
     StackProject ->
         getStackPackageDbStack
+    ExplicitProject -> return []
```

Comment:
I think you'll have to parse the options, filtering out the `--package-db`/`--package-conf` stuff to fill this in otherwise half of the autocompletetion related commands aren't going to work.